spec/rails_helper.rb

```diff
@@ -12,14 +12,14 @@
 # Headless Chrome
 require 'selenium/webdriver'
 
-Capybara.register_driver :headless_chrome do |app|
+Capybara.register_driver :selenium do |app|
   capabilities = Selenium::WebDriver::Remote::Capabilities.chrome(
-    chromeOptions: { args: %w[headless disable-gpu window-size=1366,768] }
+    chromeOptions: { args: %w[headless disable-gpu window-size=1366,2000] }
   )
   Capybara::Selenium::Driver.new app, browser: :chrome, desired_capabilities: capabilities
 end
 
-Capybara.javascript_driver = :headless_chrome
+Capybara.javascript_driver = :selenium
```

Comment:
Out of curiosity, what was failing here? The commit message said "PNGs"?

Reply:
Looks like this is addressing https://github.com/mattheworiordan/capybara-screenshot/issues/211 ?